plugin.sh

```diff
@@ -29,13 +35,14 @@ DOCKERFILE=${PLUGIN_DOCKERFILE:-Dockerfile}
 CONTEXT=${PLUGIN_CONTEXT:-$PWD}
 LOG=${PLUGIN_LOG:-info}
 EXTRA_OPTS=""
+SHOW_DIGEST="true"
 
```

Comment:
consider putting `PLUGIN_DIGEST_FILE` up here so you don't have to provide a default at every use-site?